crates/ef-testing/src/bin/fetch_dump_katana.rs

```diff
@@ -19,39 +19,33 @@ struct WorkflowRun {
     head_branch: String,
 }
 
+/// Fetches the Katana dump from Kakarot-RPC artifacts
+/// On every PR and merge, thanks to a CI job in Kakarot-RPC, the state of a Starknet devnet
+/// with all of Kakarot Cairo smart contracts deployed is dumped in an artifact called 'dump-katana'
+/// Starting a Starknet local chain from checkpoint speeds up all our processes.
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     dotenv().ok();
     let token =
         std::env::var("GITHUB_TOKEN").map_err(|_| eyre::eyre!("Missing GITHUB_TOKEN in .env"))?;
-    let url = "https://api.github.com/repos/sayajin-labs/kakarot-rpc/actions/artifacts";
+    let url = "https://api.github.com/repos/kkrt-labs/kakarot-rpc/actions/artifacts";
```

Comment:
Are you sure this is correct? [Python script](https://github.com/kkrt-labs/kakarot/blob/main/scripts/artifacts.py#L22C10-L22C77) in `kakarot` repo uses `https://api.github.com/repos/sayajin-labs/kakarot/actions/artifacts`.

Reply:
ahh, there's a permanent alias between the two, but it's safer to update to kkrt-labs in the python script as well